esphome/components/dfplayer/dfplayer.cpp

```diff
@@ -0,0 +1,56 @@
+#include "dfplayer.h"
+#include "esphome/core/log.h"
+#include <string.h>
+
+namespace esphome {
+namespace dfplayer {
+
+const uint8_t cmd_reset = 0x0c;
```

Comment:
```suggestion
const uint8_t CMD_RESET = 0x0c;
```

constant should be upper case